interface/src/app/node/node.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * "adage.gene.network" module.
+ */
+
+angular.module('adage.node', [
+  'ui.router',
+  'placeholders',
+  'ui.bootstrap'
+])
+
+.config(function config($stateProvider) {
+  $stateProvider.state('node', {
+    url: '/node/{id:int}',
+    views: {
+      main: {
+        templateUrl: 'node/node.tpl.html',
+        controller: 'NodeCtrl as ctrl'
+      }
+    },
+    data: {pageTitle: 'Node Information'}
+  });
+})
+
+.factory('NodeInfo', ['$resource', '$stateParams',
+  function($resource) {
+    return $resource('/api/v0/node/:id');
+  }
+])
+
+.factory('HeavyGenes', ['$resource', function($resource) {
+  return $resource('/api/v0/participation/');
```

Comment:
Maybe a dumb question, but I want to be sure I fully understand...

Participation tells us what genes participate in what nodes, right? Here you are defining an Angular function to query the `participation` endpoint, but calling it `HeavyGenes`. Does the API only provide access to high-weight, or heavy genes (via a filter?), or could I also get low-weight genes via this resource?

I am strongly in favor of having the naming precisely match what the feature does. If the `participation` API is filtered to only high-weight genes, then let's rename it. If the API is general, let's name the Angular service to match the back-end.